data/speakers.yml

```diff
@@ -13915,20 +13885,6 @@
   website: http://tomdale.net
   slug: tom-dale
   canonical_slug:
-- name: Tom E Enebo
-  twitter: ""
-  github: ""
-  bio: ""
-  website: ""
-  slug: tom-e-enebo
-  canonical_slug:
```

Comment:
Let's keep this one too and set the `canonical_slug` to `thomas-e-enebo` so future references point to the right record.